src/pages/documents/[id].tsx

```diff
@@ -60,44 +55,61 @@ const DocumentPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({
   return (
     <Layout title={`${document.title}`} description={getDocumentDescription(document.title)}>
       <section
-        className="mb-8 flex-1 flex flex-col"
+        className="pb-8 flex-1 flex flex-col bg-gray-100"
         data-analytics-date={family.published_date}
         data-analytics-geography={family.geography}
         data-analytics-variant={document.variant}
         data-analytics-type={document.content_type}
       >
-        <DocumentHead document={document} geography={family.geography} family={{ title: family.title, slug: family.slug }} />
+        <DocumentHead document={document} family={family} />
         {status !== "success" ? (
-          <div className="w-full flex justify-center flex-1">
+          <div className="w-full flex justify-center flex-1 bg-white">
             <Loader />
           </div>
         ) : (
-          <section className="pt-4 flex-1 flex" id="document-viewer">
+          <section className="flex-1 flex" id="document-viewer">
             <div className="container flex-1">
-              <div className="flex flex-col md:flex-row justify-between items-center pb-4 border-b border-lineBorder gap-4">
-                {passageMatches.length > 0 && (
-                  <h3>Document matches for {`'${router.query[QUERY_PARAMS.query_string]}' (${passageMatches.length})`}</h3>
+              <div className="md:flex md:h-[80vh]">
+                {hasPassageMatches && (
+                  <div className={`overflow-y-scroll pr-4 max-h-[30vh] md:block md:max-h-full ${passageClasses(document.content_type)}`}>
+                    <div className="my-4" data-cy="document-matches-description">
+                      <p className="">
+                        {passageMatches.length} matches for "<b>{`${router.query[QUERY_PARAMS.query_string]}`}</b>"
```

Comment:
What's displayed if we're browsing and `QUERY_PARAMS` is empty?

Reply:
It'll get caught at the `{hasPassageMatches && (` above this on line 73, i.e it will not attempt to render any of this section unless there are matching passages.

For future reference - passage matches are only applicable to string searches.